src/Universe.cpp

```diff
@@ -596,6 +436,8 @@ void Universe::addCell(Cell* cell) {
     log_printf(ERROR, "Unable to add Cell with ID = %d to Universe with"
                " ID = %d. Backtrace:\n%s", cell, _id, e.what());
   }
+
+  _boundaries_not_updated = true;
```

Comment:
I think it might be a bit more clear if we change from ``_boundaries_not_updated`` to something like ``_boundaries_inspected`` to avoid the double negative